### PR TITLE
Alpha launch chainspec

### PR DESCRIPTION
**Pull request by mateuszaaa**

### CODE REVIEW

**Review by iStrike7 — APPROVED** [review]
lgtm

File: rollup/node/src/command.rs [hunk 1]
```diff
@@ -106,6 +106,8 @@ impl SubstrateCli for Cli {
 			"holesky" => Box::new(chain_spec::rollup_local_config(self.randomize_chain_genesis_salt, self.chain_genesis_salt.clone(), eth_sequencers, arb_sequencers, base_sequencers, EvmChain::Holesky,
 				Some(String::from("https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frollup-holesky-rpc.gasp.xyz#/extrinsics/decode/"))
 			)),
+			"ethereum-mainnet" => Box::new(chain_spec::ethereum_mainnet( Some(String::from("https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frollup-holesky-rpc.gasp.xyz#/extrinsics/decode/"))
```
Comment (tenequm) [inline rollup/node/src/command.rs]: Why does it have this url here?

Comment (majusko) [reply rollup/node/src/command.rs:109]: looks like `"holesky" =>` config

File: rollup/runtime/src/runtime_config.rs [hunk 2]
```diff
@@ -999,15 +999,15 @@ pub mod config {
...
+			pub const IssuanceCap: Balance = 20_400_000 * DOLLARS;
+			pub const LinearIssuanceBlocks: u32 = 10_512_000u32; // 5 years
+			pub const LiquidityMiningSplit: Perbill = Perbill::from_parts(647050000); // 13'199'820
+			pub const StakingSplit: Perbill = Perbill::from_parts(235300000); // 4'800'120
+			pub const SequencerSplit: Perbill = Perbill::from_parts(117650000); // 2'400'060
 			pub const ImmediateTGEReleasePercent: Percent = Percent::from_percent(20);
-			pub const TGEReleasePeriod: u32 = 5_256_000u32; // 2 years
-			pub const TGEReleaseBegin: u32 = 100_800u32; // Two weeks into chain start
+			pub const TGEReleasePeriod: u32 = 0u32; // 2 years
+			pub const TGEReleaseBegin: u32 = 0u32; // Two weeks into chain start
```
Comment (mateuszaaa) [inline rollup/runtime/src/runtime_config.rs]: @iStrike7 please conifrm

**Review by majusko — APPROVED** [review]

**Review by tenequm — APPROVED** [review]

**Pull request merged**